source/retryable-writes/retryable-writes.rst

```diff
@@ -137,21 +137,20 @@ check the user’s command document to determine if it is a write, nor check
 whether the server is new enough to support a transaction ID for the command.
 The method should simply send the user’s command document to the server as-is.
 
-This specification does not affect write commands executed outside of a
-DriverSession or within a DriverSession where retryable writes have not been
-enabled.
+This specification does not affect write commands executed in an application
+where retryable writes have not been enabled.
```

Comment:
(This comment is for lines 134-138, but github won't let me add it there).

This contradicts the goal that from an ops point of view we want every operation (read or write) to have a sessionId.

It should not be the application's responsibility to add the sessionId when using the runCommand helper.